downloads.html

```diff
@@ -42,14 +42,15 @@ <h2>Windows</h2>
             </a>
           </li>
         </ul>
+        <p>
+          Luanti is distributed as a portable .zip archive on Windows.
+          Extract it to a folder you have write permissions, and run <code>luanti.exe</code>
```

Comment:
this is redundant, how do I extract it to a folder where I don't have write permissions?

rather I would mention the important fact that you have to extract the *entire* `luanti-5.x.x-winXX` folder